docs/docs/examples/guides/isaaclab-aica-bridge.md

```diff
@@ -0,0 +1,219 @@
+applications. This connection provides several key benefits:
+
+1. **RL Policy Testing**: AICA’s RL Policy Component SDK allows developers to deploy Reinforcement Learning (RL) models
+   directly onto real hardware through components. These models can be trained in Isaac Lab, and with AICA System
+   interacting directly with Isaac Lab, users can validate the trained policies under the same conditions in which they
+   were learned.
+
+2. **Reliable Policy Validation**: Developers can monitor the behavior of the trained policies and test the effect of
+   various parameters, enabling confident transitions from simulation to real-world deployment.
+
+3. **Digital Twin Control**: Beyond RL, running AICA System with Isaac Lab provides users with ways to interact with
+   digital twins of their robots. Applications can be authored, tested, and validated entirely in simulation before
+   connecting to actual hardware. This improves safety and enables rapid iteration in early stages, helping streamline
+   the overall development cycle.
+
+With Isaac Lab as simulator, users can build complete automation pipelines in Isaac Lab, interact with them using AICA
+System, validate performance, switch the hardware interface to a real robot, and hit play with no code changes required.
+
+# Installing Isaac Lab
+
+Begin by cloning AICA's fork of [Isaac Lab](https://github.com/aica-technology/isaac-lab).
+
+Once the repository is cloned, build and start the Docker container by running:
+
+```shell
+python3 docker/container.py start
+```
+
+This command will build the Docker image and start the container in the background.
+
+Next, enter the running container using:
+
+```shell
+python3 docker/container.py enter
+```
+
+This ensures you are inside a development environment where Isaac Lab and all required dependencies are already
+installed.
+
+Once done, verify the installation by running
+
+```shell
+python3 scripts/custom/aica_bridge/run_bridge.py --scene basic_scene
+```
+
+This will spawn a UR5e robot, a ground plane and lights. If you see the UR5e robot in the scene, then the installation
+was successful and you are ready to proceed with the next steps.
+
+If not, carefully go over the instructions again or reach out to our [support email](contact@aica.tech) for help.
+
+# Connecting AICA System to Isaac Lab
+
+In this section, we’ll walk through a simple example of using a point attractor to move a UR5e simulated robot in Isaac
+Lab to a target frame and then manipulate that frame in the 3D visualization of AICA Studio.
+
+## Creating a New Scene in Isaac Lab
+
+To create a new scene, you should define a scene config class that inherits from InteractiveSceneCfg. Various examples
+of scene config classes can be found in the [`scenes`](https://github.com/aica-technology/isaac-lab/blob/main/scripts/custom/aica_bridge/scenes) directory of the Isaac Lab
```

Comment:
for the links, I think we should reference a tag on that repo (which means we should also create one), otherwise we risk breaking the docs by making changes upstream